spec/features/login_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+RSpec.describe 'Login', type: :feature, ui: true do
+  let(:login_path) { '/en/session/new' }
+  let(:profile_path) { '/en/profile' }
+
+  before { visit login_path }
+
+  describe 'as a standard user' do
+    before { create :user }
+
+    before do
+      fill_inputs 'session', params.slice(:email, :password)
+    end
+
+    context 'with everything valid' do
+      let(:params) { attributes_for :user }
+
+      before { click_button 'commit' }
+
+      it('redirect to profile page') { expect(page).to have_current_path(profile_path) }
+      it { expect(page).to have_field('user_name', with: params[:name]) }
+      it { expect(page).to have_field('user_username', with: params[:username]) }
+      it { expect(page).to have_field('user_city', with: params[:city]) }
+      it { expect(page).to have_field('user_institution', with: params[:institution]) }
+      it { expect(page).to have_link('Ask for confirmation') }
+      it { expect(page).to have_no_link('Workers', visible: :all) }
+    end
+
+    context 'with wrong email' do
+      let(:params) { attributes_for :user, email: 'judy.doe@codelabs.test' }
+
+      before { click_button 'commit' }
+
+      it('redirect to login page') { expect(page).to have_current_path(login_path) }
+      it { expect(page).to have_text('Email is invalid') }
+    end
+
+    context 'with wrong password' do
+      let(:params) { attributes_for :user, password: SecureRandom.base36 }
+
+      before { click_button 'commit' }
+
+      it('redirect to login page') { expect(page).to have_current_path(login_path) }
+      it { expect(page).to have_text('Password is invalid') }
+    end
+  end
+
+  describe 'as an admin user' do
+    before { create :admin_user }
```

Comment:
Why not to do it like this? Then there'll be no need for `admin_user` factory.

```suggestion
    before { create :user, :admin }
```

Reply:
fixed